src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -40,7 +40,7 @@ def _get_client(service, parsed_args):
     account_name = parsed_args.account_name or os.getenv('AZURE_STORAGE_ACCOUNT')
     account_key = parsed_args.account_key or os.getenv('AZURE_STORAGE_KEY')
     connection_string = parsed_args.connection_string or os.getenv('AZURE_STORAGE_CONNECTION_STRING')
-    sas_token = parsed_args.sas_token or os.getenv('AZURE_SAS_TOKEN')
+    sas_token = parsed_args.sas_token or os.getenv('AZURE_STORAGE_SAS_TOKEN')
```

Comment:
Hmm...missed these. I was looking for instances of os.env.get. I'll update these too.

---
In reply to: [77908310](https://github.com/Azure/azure-cli/pull/804#discussion_r77908310) [](ancestors = 77908310,77391940)